setup.py

```diff
@@ -13,4 +13,42 @@
 
 import setuptools
 
-setuptools.setup()
+
+def load_requirements(*requirements_paths):
+    """
+    Load all requirements from the specified requirements files.
+
+    Returns:
+        list: Requirements file relative path strings
+    """
+    requirements = set()
+    for path in requirements_paths:
+        requirements.update(
+            line.split("#")[0].strip()
+            for line in open(path).readlines()
+            if is_requirement(line.strip())
+        )
+    return list(requirements)
+
+
+def is_requirement(line):
+    """
+    Return True if the requirement line is a package requirement.
+
+    Returns:
+        bool: True if the line is not blank, a comment, a URL, or an included file
+    """
+    return not (
+        line == ""
+        or line.startswith("-r")
+        or line.startswith("#")
+        or line.startswith("-e")
+        or line.startswith("git+")
+        or line.startswith("-c")
+    )
+
+
+setuptools.setup(
+    include_package_data=True,
+    install_requires=load_requirements("requirements/base.in"),
+)
```

Comment:
I'm pretty lazy when it comes to reviewing. I didn't realize you were just copying this code from cookiecutter.  Is that correct, or do I need to carefully review this?

Reply:
This comes from [edx-toggles' setup.py](https://github.com/edx/edx-toggles/blob/master/setup.py) but I'll copy this from the [cookiecutter](https://github.com/edx/edx-cookiecutters/blob/master/python-template/%7B%7Bcookiecutter.placeholder_repo_name%7D%7D/setup.py) instead.